CHANGELOG.md

```diff
@@ -1,11 +1,115 @@
+* MVP for soft opt out, see [EPIC](https://github.com/cosmos/interchain-security/issues/851)
+* Various fixes, critical and non-critical
+* Docs updates which should not affect production code
+
+## PRs included in v1.3.0
+
+* (fix) Prevent denom DOS [#931](https://github.com/cosmos/interchain-security/pull/931)
+* (fix) multisig for assigning consumer key, use json [#916](https://github.com/cosmos/interchain-security/pull/916)
+* (feat) Add warnings when provider unbonding is shorter than consumer unbonding [#858](https://github.com/cosmos/interchain-security/pull/858)
+* (chore) use go 1.19 [#899](https://github.com/cosmos/interchain-security/pull/899)
+* (feat) Standalone to consumer changeover - recycle existing transfer channel [#832](https://github.com/cosmos/interchain-security/pull/832)
+* (deps) Bump IBC [862](https://github.com/cosmos/interchain-security/pull/862)
+* (chore) Use go 1.20 [#840](https://github.com/cosmos/interchain-security/pull/840)
+* (feat) Standalone to consumer changeover - staking functionalities [#794](https://github.com/cosmos/interchain-security/pull/794)
+* (fix) prevent provider from sending VSCPackets with multiple updates for the same validator [#850](https://github.com/cosmos/interchain-security/pull/850)
+* (feat) Soft opt out [#833](https://github.com/cosmos/interchain-security/issues/833)
+* (fix) Correctly handle VSC packet with duplicate val updates on consumer [#846](https://github.com/cosmos/interchain-security/pull/846)
+* (deps) bump sdk to v0.45.15.ics [#805](https://github.com/cosmos/interchain-security/pull/805)
+* (refactor) Remove spm module [#812](https://github.com/cosmos/interchain-security/pull/812)
+* (feat) Standalone to consumer changeover part 1 [#757](https://github.com/cosmos/interchain-security/pull/757)
+* (chore) Swap names of e2e and integration tests [#681](https://github.com/cosmos/interchain-security/pull/681)
+* (fix) fix StopConsumerChain not running in cachedContext [#802](https://github.com/cosmos/interchain-security/pull/802). Also in earlier releases with different commit order!
+* (docs) Introduce docs website [#759](https://github.com/cosmos/interchain-security/pull/759)
+* (fix) Ser correct byte prefix for SlashLogKey [#786](https://github.com/cosmos/interchain-security/pull/786)
+* (feature) Improve keeper field validation [#766](https://github.com/cosmos/interchain-security/pull/766)
+* (docs) Contributing guidelines [#744](https://github.com/cosmos/interchain-security/pull/744)
+* (refactor) Key assignment type safety [#725](https://github.com/cosmos/interchain-security/pull/725). Note this is technically a refactor but is a large enough change to be included in changelog
+* (fix) Update protos and fix deps [#752](https://github.com/cosmos/interchain-security/pull/752)
+* (api) Add consumer QueryParams [#746](https://github.com/cosmos/interchain-security/pull/746)
+* (fix) Nits from audit [#743](https://github.com/cosmos/interchain-security/pull/743)
```

Comment:
If they are nits, we should remove them. Otherwise, we should be more explicit about the change.

Reply:
They were nits that shouldn't meaningfully affect the protocol, removed